borg/archiver.py

```diff
@@ -1458,10 +1419,8 @@ def setup_signal_handlers():  # pragma: no cover
 
 
 def main():  # pragma: no cover
-    # Make sure stdout and stderr have errors='replace') to avoid unicode
-    # issues when print()-ing unicode file names
-    sys.stdout = io.TextIOWrapper(sys.stdout.buffer, sys.stdout.encoding, 'replace', line_buffering=True)
-    sys.stderr = io.TextIOWrapper(sys.stderr.buffer, sys.stderr.encoding, 'replace', line_buffering=True)
+    sys.stdout = io.TextIOWrapper(sys.stdout.buffer, sys.stdout.encoding, 'surrogateescape', line_buffering=True)
+    sys.stderr = io.TextIOWrapper(sys.stderr.buffer, sys.stderr.encoding, 'surrogateescape', line_buffering=True)
```

Comment:
I don't think this is necessary anymore (at all) - maybe for stderr, but I think all places use remove_surrogates when printing to stdout.